java/client/pom.xml

```diff
@@ -7,7 +7,6 @@
     <version>1.0-SNAPSHOT</version>
     <relativePath>../pom.xml</relativePath>
   </parent>
-  <groupId>com.youtube.vitess</groupId>
```

Comment:
Maven 3.3.3 was giving me a warning saying something to the effect of "you should omit groupId if it's the same as the groupId of the parent"